ansible/templates/pm4ml.values.yml.j2

```diff
@@ -217,8 +250,9 @@ ttk:
           host: ttk-specapi.{{ item.key }}.{{ item.value.pm4ml_subdomain }}
         adminApi:
           host: ttk.{{ item.key }}.{{ item.value.pm4ml_subdomain }}
-      className: {{ pm4ml_ingress_classname }}
+      className: null

```

Comment:
why are you reverting the ingress class changes @vijayg10 ?

Reply:
Hi David, className param is not working because of the k8s version used in pm4ml due to the helm chart implementation here https://github.com/mojaloop/helm/blob/4e28b8edf15cc02d5d8dffc4340ecffba75513d4/ml-testing-toolkit/chart-backend/templates/ingress.yaml#L18

So I am setting the className to null and specifying the ingress class through annotations just like the other services.

Reply:
@dfry